lib/manager/components/GeneralSettings.js

```diff
@@ -168,6 +193,34 @@ export default class GeneralSettings extends Component<Props, State> {
                 </Checkbox>
                 <small>Set this feed source to fetch automatically. (Feed source URL must be specified and project auto fetch must be enabled.)</small>
               </FormGroup>
+              {autoFetchFeed
```

Comment:
This appears to be the same as the component in the `CreateFeedSource`. Seems like the fetch feed frequency could be its own component that take in handler functions for changes in frequency and intervals.